packages/token-bridge-sdk/src/deposits/fetchDepositsFromSubgraph.ts

```diff
@@ -57,44 +58,36 @@ export const fetchDepositsFromSubgraph = async ({
     return []
   }
 
-  const res = await getL1SubgraphClient(l2ChainId).query({
-    query: gql`{
-        deposits(
-          where: {
-            sender: "${address}",
-            blockCreatedAt_gte: ${fromBlock},
-            blockCreatedAt_lte: ${toBlock}
-            ${searchString ? `transactionHash_contains: "${searchString}"` : ''}
-          }
-          orderBy: blockCreatedAt
-          orderDirection: desc
-          first: ${pageSize},
-          skip: ${pageNumber * pageSize}
-        ) {
-          receiver
-          sender
-          sequenceNumber
-          timestamp
-          tokenAmount
-          transactionHash
-          type
-          isClassic
-          id
-          ethValue
-          blockCreatedAt
-          l1Token {
-            symbol
-            decimals    
-            id
-            name
-            registeredAtBlock
-          }                  
-        }
-      }
-    `
-  })
+  // if dev environment, eg. tests, then prepend actual running environment
+  // Resolves: next-js-error-only-absolute-urls-are-supported in test:ci:sdk
+  const baseUrl =
+    window.location.origin.indexOf('localhost') > -1
+      ? 'http://localhost:3000'
+      : window.location.origin
```

Comment:
We should be able to check `NODE_ENV`: https://jestjs.io/docs/environment-variables#node_env

```suggestion
  const baseUrl =
    process.env.NODE_ENV === 'test'
      ? 'http://localhost:3000'
      : ''
```

For non-test env, we should use relative paths

Reply:
Done. Passing the test in local sdk test. Awaiting results here.

Reply:
SDK test passing perfectly in ci as well. Nice.